cogboard-webapp/src/components/Widget/styled.js

```diff
@@ -89,6 +89,21 @@ StyledCard.defaultProps = {
 export const StyledCardHeader = styled(CardHeader)`
   z-index: 1;
   padding: 8px;
+
+  .MuiCardHeader-action,
+  .MuiSvgIcon-root {
```

Comment:
There is another way to select these elements and it will be more consistent with the rest of the codebase: https://material-ui.com/guides/interoperability/#deeper-elements.
Please use aliases for these classnames.

Reply:
Done.